index2.html

```diff
@@ -0,0 +1,360 @@
+              <li class="html">HTML</li>
+            </ul>
+          </div>
+          <button type="submit" class="see-project-btn">See Project</button>
+        </div>
+      </div>
+      <div class="container">
+        <div class="column1"></div>
+        <div class="column2">
+          <div class="project-box">
+            <h1 class="project-name1">Multi-Post Stories</h1>
+            <h1 class="project-name-two">Gain+Glory</h1>
+          </div>
+          <div class="languages">
+            <ul>
+              <li class="ruby">Ruby on Rails</li>
+              <li class="css">css</li>
+              <li class="js">JavScript</li>
+              <li class="html">HTML</li>
+            </ul>
+          </div>
+          <button type="submit" class="see-project-btn">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="about-me">About Me</div>
+      <img class="rocket" alt="" src="rocket.svg" />
+    </section>
+    <section class="text-box1">
+      <div class="text-info1">
+        <h2 class="text1">I can help you build a product,</h2>
+        <h2 class="text2">feature or website Look through</h2>
+        <h2 class="text3">some of my work and experience! If</h2>
+        <h2 class="text4">you like what you see and have a</h2>
+        <h2 class="text5">project you need coded,</h2>
+        <h2 class="text6">don’t hestiate to contact me.</h2>
+      </div>
+    </section>
+
+    <section class="text-box2">
+      <div class="text-info2">
+        <h2 class="text1a">
+          I can help you build a product , feature or website Look through some
+          of my work and
+        </h2>
+        <h3 class="text-info2b">
+          experience! If you like what you see and have a project you need
+          coded,
+        </h3>
+        <h4 class="text-info3c">don’t hestiate to contact me.</h4>
+      </div>
+    </section>
+
+    <button type="submit" class="resume-btn">Get My Resume</button>
+    <section class="end-section">
+      <div class="big-bit">
+        <div class="language-container">
+          <div class="language-column">
+            <h1 class="languages2">Languages</h1>
+            <img class="globe" alt="" src="ball.svg" />
+          </div>
+        </div>
+        <div class="center-row">
+          <ul>
+            <li class="java-language">Javascript</li>
+            <li class="ruby-language">Ruby</li>
+            <li class="html-language">HTML</li>
+            <li class="css-language">CSS</li>
+          </ul>
+        </div>
+
+        <div class="language-container">
+          <div class="language-column">
+            <h1 class="frameworks2">FrameWorks</h1>
+            <img class="frame2" alt="ficon" src="frame.svg" />
+          </div>
+          <div class="top-row">
+            <ul>
+              <li class="boot">Bootstrap</li>
+              <li class="ruby-rails">Ruby on rails</li>
+              <li class="r-spec">RSpec</li>
```

Comment:
- [ ] Great job 👍 . Kindly fix the color contrast issue of all the text in your `li` tag located in your `About Me` section 🙏

![image](https://user-images.githubusercontent.com/74814780/188691925-83efa82c-6f72-407e-8ba2-1504575cc67b.png)

Reply:
Hi @Ugboaja-Uchechi  i changed the color to #477e7e